maven/plugins/com.liferay.ide.maven.core/plugin.xml

```diff
@@ -43,6 +43,16 @@
       </liferayProjectProvider>
    </extension>
    <extension
+         point="com.liferay.ide.core.liferayProjectProviders">
```

Comment:
Lets not create a new project provider, lets add it to the existing maven project provider.  I'm ok with one provider knowing how to create multiple project types.